### PR TITLE
Change structure

### PR DESCRIPTION
**Pull request by outcatcher**
Move to pub/sub approach for inputs and outputs communication

### CODE REVIEW

File: delatore/sources/influx.py [hunk 1]
```diff
@@ -92,9 +92,8 @@ def __init__(self, *args, **kwargs):
         if isinstance(data, (dict, list)):
             data = json.dumps(data)
 
-        # Try to send the request more than once by default (see #103)
-        retry = True
-        _try = 0
+        # retry = True
+        # _try = 0
```
Comment (anton-sidelnikov) [inline delatore/sources/influx.py]: remove it if not use

**Comment by outcatcher** [comment]
Still need real acceptance test

File: delatore/sources/awx_api.py [hunk 2]
```diff
@@ -21,48 +25,62 @@ class NoSuchTemplate(Exception):
...
 
+TIMESTAMP_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+"""Timestamp format with milliseconds"""
+
+
 class TemplateStatus(NamedTuple):
     name: str
     last_run_timestamp: str
     last_status: str
     playbook: str
```
Comment (anton-sidelnikov) [inline delatore/sources/awx_api.py]: never used

Comment (outcatcher) [reply delatore/sources/awx_api.py]: Used in _status_json

Comment (outcatcher) [reply delatore/sources/awx_api.py]: TIMESTAMP_FORMAT used in `__md__` and in tests

**Comment by outcatcher** [comment]
Acceptance test passed

**Pull request merged**